support/Python/requirements.txt

```diff
@@ -16,6 +16,8 @@
 # Click: to compose our Python CLI, load subcommands lazily, and for
 # autocompletion
 click
+# GitPython: to interact with Git repositories, e.g. SpEC version check
+GitPython
```

Comment:
Let's not continue ballooning our dependencies. It's just a git call. See my comment in the python file for how to do this.